### PR TITLE
How to create an Apple PassType ID

### PR DESCRIPTION
**Pull request by grundyoso**
A hardware guide for creating a PassType ID to work up to the "How to create an NFC car key" guide.

### CODE REVIEW

File: app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+title: How to Create an Apple Pass Type
+author: @grundy
+
+## Overview
+
+This guide is designed to help developers quickly generate pass types on Apple's Developer portal. A pass type consititutes a unique identifier and a signing certificate. Whether you're a seasoned developer or a nooB, this guide will get you through the steps like a ninja. We start with creating a pass type identifier for your organization and group of passes you want to distribute. This identifier, in the form of a reverse DNS string, is created in the Identifiers section of your Apple Developer portal. Once you have the pass type id, you'll need a signing certificate to go along with it. You get the signing certificate by creating a certificate signing request (CSR) on your PC and then generating the certificate in the Certificates section of the same portal. Once generated, you then can download the signing certificate to your machine for signing a pass bundle.
```
Comment (abunsen) [inline app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md:6]:
```suggestion
This guide is designed to help developers quickly generate pass types on Apple's Developer portal. A pass type consititutes a unique identifier and a signing certificate. Whether you're a seasoned developer or a beginner, this guide will get you through the steps like a ninja. 

We start with creating a pass type identifier for your organization and group of passes you want to distribute. This identifier, in the form of a reverse DNS string, is created in the Identifiers section of your Apple Developer portal. Once you have the pass type id, you'll need a signing certificate to go along with it. You get the signing certificate by creating a certificate signing request (CSR) on your PC and then generating the certificate in the Certificates section of the Apple developer portal. 

Once generated, you then can download the signing certificate to your machine for signing a pass bundle.
```

File: app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Create Pass Type ID
+
+To start you'll need to create a pass type identifier in your Apple Developer account. This is done by filling out a simple form under the Identifiers section. The identifier should be a unique string, without spaces, typically following a reverse DNS pattern so that sorting a list of many identifiers groups them by organization instead of application.
+
+### Go to [Apple Developer portal](https://developer.apple.com/)
```
Comment (abunsen) [inline app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md:12]:
```suggestion
**Go to [Apple Developer portal](https://developer.apple.com/)**
```

File: app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+![step1](tutorials/how-to-create-a-apple-pass-type-id/step1.jpg)
+
+### Sign in to your developer account
```
Comment (abunsen) [inline app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md:16]:
```suggestion
**Sign in to your developer account**
```

File: app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+![step2](tutorials/how-to-create-a-apple-pass-type-id/step2.jpg)
+
+### Select Identifiers and click Add (+)
```
Comment (abunsen) [inline app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md:20]:
```suggestion
**Select Identifiers and click Add (+)**
```

File: app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+![step3](tutorials/how-to-create-a-apple-pass-type-id/step3.jpg)
+
+### Create a Pass Type Id
+
+![step4](tutorials/how-to-create-a-apple-pass-type-id/step4.jpg)
+
+### Register your Identifier with Description
```
Comment (abunsen) [inline app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md:28]:
```suggestion
**Register your Identifier with description**
```

File: app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+![step5](tutorials/how-to-create-a-apple-pass-type-id/step5.jpg)
+
+## Generate Signing Certificate
+
+Once a pass type identifier is registered on the Apple Developer portal, you'll need a certificate to start using it in making Apple Wallet passes. Similar to how application builds (.ipa files) need to be cryptographically signed by their developer before they can be installed on devices, Apple pass bundles (.pkpass files) need to be signed by their issuer before they can be installed in Apple Wallets. So you will need a signing certificate for each pass type identifier you create and these certificates expire every 398 days.
+
+### Prepare a Certificate Signing Request
```
Comment (abunsen) [inline app/views/tutorial-files/apple-platform/how-to-create-an-apple-pass-type-id.md:36]:
```suggestion
**Prepare a Certificate Signing Request**
```

**Review by abunsen — DISMISSED** [review]
Made a bunch of suggestions, you can literally click "commit suggestion" on every one that you agree with. If you accept them all, then we're done :D

**Comment by grundyoso** [comment]
Made all the requested changes. Added further word smithing and typo fixes I found. Also cleaned up the PR to only include the work done on this branch (had to revert the "dirty commit" 6ddc9bd747f4a57db866afed070023f22e3f180b that was brought in by the cherry-pick). Please review and merge pull request.

**Pull request merged**